src/map/handler/Map.TapHold.js

```diff
@@ -0,0 +1,108 @@
+import {Map} from '../Map';
+import {Handler} from '../../core/Handler';
+import * as DomEvent from '../../dom/DomEvent';
+import {Point} from '../../geometry/Point';
+import * as Util from '../../core/Util';
+import * as Browser from '../../core/Browser';
+
+/*
+ * L.Map.TapHold is used to simulate `contextmenu` event on long hold,
+ * which otherwise is not fired by mobile Safari.
+ */
+
+var tapHoldDelay = 600;
+
+// @namespace Map
+// @section Interaction Options
+Map.mergeOptions({
+	// @section Touch interaction options
+	// @option tapHold: Boolean
+	// Enables simulation of `contextmenu` event, default is `true` for mobile Safari.
+	tapHold: Browser.touch && Browser.safari && Browser.mobile,
```

Comment:
I didn't found a usage in the code or is it that one `Map.addInitHook('addHandler', 'tapHold', TapHold);`?

Reply:
https://github.com/Leaflet/Leaflet/blob/5516f8ae5308142dc3fc75633f15abfc6dc28b2f/src/map/Map.js#L727

Reply:
Ok interessting